gym/spaces/graph.py

```diff
@@ -84,6 +19,8 @@ class Graph(Space):
         self.observation_space = spaces.Graph(node_space=space.Box(low=-100, high=100, shape=(3,)), edge_space=spaces.Discrete(3))
     """
 
+    _graph_obj_ctor = namedtuple("graph_obj", ["nodes", "edges", "edge_links"])
```

Comment:
Normally, NamedTuples are specified in global scope like a normal class. You could call it `GraphInstance`?